Index.html

```diff
@@ -11,18 +11,22 @@
 
 <body>
     <header>
-        <nav>
-            <div class="navContainer">
-                <a href="#"><span class="nick">Elio Nel</span></a>
-                <img src="./icons/IconMenu.png" alt="menu icon">
-            </div>
+        <nav class="navContainer">
+                 <a href="#"><span class="nick">Elio Nel</span></a>
+                <img class="hamburguer" src="./icons/IconMenu.png" alt="menu icon">
+                <ul class="menu">
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                    <li><a href="#"><img src="./icons/message.png" alt="msg icon"></a></li>
+                </ul>
```

Comment:
Your design             |  Figma Design
:-------------------------:|:-------------------------:
![tse1](https://user-images.githubusercontent.com/69638013/155658145-37a6d066-cb74-4cbd-93b5-d38608966716.PNG)  |  ![nav2](https://user-images.githubusercontent.com/69638013/155658169-e79249d6-d53d-433e-b68d-7a7ea3c88e71.PNG)

- [ ] Kindly align nav links to the right check screen shot above 👍🏿